cocotb/share/makefiles/Makefile.inc

```diff
@@ -97,6 +97,8 @@ else
 $(error "Unsupported os " $(OS))
 endif
 
+GXX_ARGS += -std=c++11 -Wsuggest-override
```

Comment:
GCC 4.8.5 is supported by cocotb and doesn't support `-Wsuggest-override`, so this needs to be removed. (https://github.com/cocotb/cocotb/pull/1149#issuecomment-560113536)

Reply:
We need this flag in place for useful CI, but agreed that we should support older GCC - I'll try to find a better place for it.

Reply:
Tried to make this conditional on the compiler version